main.py

```diff
@@ -1,22 +1,37 @@
-from flask import Flask, request
-from flask_json import FlaskJSON
+import json
+from flask import Flask, request, make_response
 
 app = Flask(__name__)
-FlaskJSON(app)
 
-@app.route("/")
-def hello():
-    return "Hello World!"
+class ContentError(Exception):
+    pass
+class LogError(Exception):
+    pass
 
 @app.route('/login', methods=['POST'])
 def login():
+    try:
+        if request.headers['Content-Type'] == 'application/json':
```

Comment:
Despite what I said earlier in IM, Flask can handle some of the checks for you and still enable you to return an error.
https://flask.palletsprojects.com/en/1.1.x/api/#flask.Request.get_json